### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by sappenin**
Update README for v1.0 release

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -88,21 +88,56 @@ In order to properly build this project, you must download and install Java Cryp
...
 
 #### Maven
+Modules in this library can be included in your Maven project by adding the Maven dependency for the module you would like to use. For example, to import `ilp-core`, use the following coordinates:
+
+```
+<dependencies>
+  ...
+  <dependency>
+    <groupId>org.interledger</groupId>
+    <artifactId>java-ilp-core</artifactId>
```
Comment (nhartner) [inline README.md]: artifactId is `ilp-core`

File: README.md [hunk 2]
```diff
@@ -88,21 +88,56 @@ In order to properly build this project, you must download and install Java Cryp
...
+</dependencies>
+```
+
+#### Gradle
+Modules in this library can be included in your Gradle project by adding the Maven dependency for the module you would like to use. For example, to import `ilp-core`, use the following coordinates:
+
+```
+dependencies {
+    ...
+    compile group: 'org.interledger', name: 'java-ilp-core', version: '1.0'
```
Comment (nhartner) [inline README.md]: `ilp-core`

File: README.md [hunk 3]
```diff
@@ -111,13 +146,13 @@ Next, add the following Maven dependency:
   <dependency>
     <groupId>org.interledger</groupId>
     <artifactId>java-ilp-core</artifactId>
```
Comment (nhartner) [inline README.md]: Change to just ilp-core

File: README.md (hunk 1 above)
Comment (sappenin) [reply README.md]: Fixed.

File: README.md (hunk 3 above)
Comment (sappenin) [reply README.md]: Fixed.

File: README.md (hunk 2 above)
Comment (sappenin) [reply README.md]: Fixed.

**Pull request merged**